testing/get-lua.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# Install the runtime.
+sudo apt install lua5.3 liblua5.3-dev
+# Ubuntu doesn't add a `lua` binary. Symlink to make the test config easier.
+sudo ln -s /usr/bin/lua5.3 /usr/bin/lua
```

Comment:
I don't feel great about this line. An Linux user seems unlikely to symlink a specific version to `lua` so this is setting up the test environment for an unrealistic circumstance.

I could update the code to search every directory in the PATH and attempt to find `lua*`. This becomes even more relevant since I had to add in some code that searches for the version number to put in `LUA_PATH` and `LUA_CPATH`. That part of this branch feels brittle to me at the moment and I'm open to suggestions to improve it.

Reply:
yeahhhh this is not a good idea, at the very least it should be `/usr/local/bin/lua` -- but it also strikes me as odd that there isn't a non-versioned executable -- then again I haven't written much lua

Reply:
maybe pre-commit should try and discovery any executable that looks like `lua-#.#`?  what do lua developers usually do?

Reply:
> what do lua developers usually do?

I have no idea. I'm fairly new to Lua myself. I'll try to make the script do some discovery.